src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
+    const newValue = key;
```

Comment:
You can use Object.keys / Object.values instead of this loop. And this is better for me not to declare variables inside the loop.